installation/sql/postgresql/base.sql

```diff
@@ -397,6 +398,7 @@ INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder",
 (0, 'plg_webservices_contact', 'plugin', 'contact', 'webservices', 0, 1, 1, 0, 1, '', '{}', '', 3, 0),
 (0, 'plg_webservices_content', 'plugin', 'content', 'webservices', 0, 1, 1, 0, 1, '', '{}', '', 4, 0),
 (0, 'plg_webservices_installer', 'plugin', 'installer', 'webservices', 0, 1, 1, 0, 1, '', '{}', '', 5, 0),
+(0, 'plg_webservices_joomlaupdate', 'plugin', 'joomlaupdate', 'webservices', 0, 1, 1, 0, 1, '', '{}', '', 5, 0),
```

Comment:
```suggestion
(0, 'plg_webservices_joomlaupdate', 'plugin', 'joomlaupdate', 'webservices', 0, 1, 1, 0, 1, '', '{}', '', 6, 0),
```
Should be the same as in the MySQL file.

The same comment also applies here, the ordering needs to be cleaned up so it is unique for all the webservices plugins.

But that can be done with a follow up PR.

Reply:
See https://github.com/SniperSister/joomla-cms/pull/4 for fixing this.